src/detect-engine-enip.c

```diff
@@ -328,7 +328,7 @@ static int DetectEngineInspectENIPTest01(void)
     p->flags        |= PKT_HAS_FLOW | PKT_STREAM_EST;
     p->flowflags    |= FLOW_PKT_TOSERVER | FLOW_PKT_ESTABLISHED;
 
-    StreamTcpInitConfig(TRUE);
+    StreamTcpInitConfig(true);
```

Comment:
wonder why this is not covered by tests. Are these unittests not executed somehow?

Reply:
Register's never invoked ... I'll update this too.